tests/simple/testdata/block_ext.textproto

```diff
@@ -557,42 +557,42 @@ section: {
   }
   test: {
     name: "multiple_macros_1"
-    expr: "cel.@block([[1], cel.iterVar(0, 0) > 0, cel.accuVar(0, 0) || @index1, [2], cel.iterVar(0, 0) > 1, cel.accuVar(0, 0) || @index4], size([@index0.exists(cel.iterVar(0, 0), @index1)]) + size([@index0.exists(cel.iterVar(0, 0), @index1)]) + size([@index3.exists(cel.iterVar(0, 0), @index4)]) + size([@index3.exists(cel.iterVar(0, 0), @index4)]) == 4)"
+    expr: "cel.block([[1], cel.iterVar(0, 0) > 0, cel.accuVar(0, 0) || @index1, [2], cel.iterVar(0, 0) > 1, cel.accuVar(0, 0) || @index4], size([@index0.exists(cel.iterVar(0, 0), @index1)]) + size([@index0.exists(cel.iterVar(0, 0), @index1)]) + size([@index3.exists(cel.iterVar(0, 0), @index4)]) + size([@index3.exists(cel.iterVar(0, 0), @index4)]) == 4)"
```

Comment:
do the `@index` also need to be `cel.index`?

Reply:
Thanks. lol Done.